git-branchless-lib/src/core/rewrite/plan.rs

```diff
@@ -148,7 +148,65 @@ impl ToString for RebaseCommand {
             } => match commits_to_apply_oids.as_slice() {
                 [] => String::new(),
                 [commit_oid] => format!("pick {commit_oid}"),
-                [..] => unimplemented!("On-disk fixups"),
+                [pick_oid, fixup_oids @ ..] => {
+                    let mut picks = vec![format!("pick {pick_oid}")];
+                    let mut fixups = fixup_oids
+                        .iter()
+                        .map(|oid| format!("fixup {oid}"))
+                        .collect::<Vec<String>>();
+                    let mut cleanups = vec![];
+
+                    // Since 0ca8681, the intermediate commits created as each
+                    // fixup is applied are left behind in the smartlog. This
+                    // forcibly removes all but the last of them. I don't
+                    // understand why this happens during `git branchless`
+                    // initiated rebases, but not during "normal" fixup rebases,
+                    // but this makes these artifacts go away.
+                    if fixups.len() > 1 {
+                        fixups = intersperse(
+                            fixups,
+                            // FIXME Is $(...) portable?
```

Comment:
I was thinking about how it would work on Windows, given that this will be using `$(...)` constructs in the rebase plans for on-disk rebases. Of course, the Windows tests are working, so I'm probably worrying over nothing. Regardless, I'll remove this comment. Thanks!